docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/media/video/video_macros.twig

```diff
@@ -14,7 +14,7 @@
       {# Check whether the active video display mode has explicitly set the 'name' as an available field #}
       {% set hasTitle = content.name %}
       {% if hasTitle %}
-        <h4 tabindex="0" class="title">{{ name }}</h4>
+        <span tabindex="0" class="video-title__heading">{{ name }}</span>
```

Comment:
You should not be changing this macro. If this macro **IS** being called, then you should fix all the templates that reference this file.